webroot/css/bootstrap-u-i.css

```diff
@@ -0,0 +1,13 @@
+.form-group.date .form-control,
+.form-group.time .form-control,
+.form-group.datetime .form-control {
+    width: auto;
+    display: inline-block;
+    margin-right: 6px;
+    margin-bottom: 2px;
+}
+
+.form-horizontal .checkbox, .form-horizontal .checkbox-inline, .form-horizontal .radio, .form-horizontal .radio-inline {
+    padding-top: 0px;
+    margin-bottom: 4px;
+}
```

Comment:
@jadb This CSS file was still valid IMO to make the styling of the default elements better.